server_commands/moderation.py

```
@@ -14,16 +25,13 @@ async def announce(self, ctx, message: str):
     @on("member_join")
     async def greet_member(self, member):
         """DM new members a welcome message."""
         try:
-            await member.send(
-                f"👋 Welcome to **{member.guild.name}**, {member.name}!\n"
-                "Feel free to introduce yourself."
-            )
+            await member.send(_welcome_message(member))
         except Exception:
-            pass  # DMs may be disabled
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)